### DISCUSSION TITLE
Bill repeats question

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6554] [Firefly III]
I have a bill that repeats every 80 days, how can I set this in the Bill?
I thought about setting it to daily and then skip 79 days but the max value here is 31 so now not sure.

**Comment by JC5** [comment]
Sorry no. I'm interested though, what kind of bill hits every 80 days?

**Reply by Solverz-0** [reply]
It is a bill for cosmetics, they give you a supply every 80 days so the payment falls on the 80th day each period.

Odd and frustrating I know...

**Reply by Solverz-0** [reply]
Do you have any alternative work arounds you would suggest?

**Reply by JC5** [reply]
For now, set it to 11 weeks. That's the closest you'll get. I'll see what I can do in the meantime. Optionally, you can also update the start date of the bill every now and then.

**Reply by Solverz-0** [reply]
Okay thank you, as a workaround I will set it to 11 weeks, hopefully this is possible in future updates :) 
Appreciate your help and this amazing software!
